### PR TITLE
fix: class duplication error

### PR DESCRIPTION
**Pull request by jakobuid**
- Fix for a class duplication error
- Update minor dependencies
- Remove duplicated dependencies in gradle files

### CODE REVIEW

File: buildSrc/src/main/kotlin/Dependencies.kt [hunk 1]
```diff
@@ -128,7 +128,7 @@ fun DependencyHandlerScope.androidXTest() {
     "testImplementation"("androidx.test:core-ktx:$androidxTestVersion")
 
     "androidTestImplementation"("androidx.test:core-ktx:$androidxTestVersion")
-    "androidTestImplementation"("androidx.test:runner:$androidxTestVersion")
+    "androidTestImplementation"("androidx.test:runner:1.5.2")
```
Comment (Elyniss) [inline buildSrc/src/main/kotlin/Dependencies.kt:131]: Why is here a specific value?

Comment (jakobuid) [reply buildSrc/src/main/kotlin/Dependencies.kt:131]: Because the newest version for runners differs from the core one

**Review by Elyniss — APPROVED** [review]

File: androidCore/sample/build.gradle.kts [hunk 2]
```diff
@@ -19,6 +19,8 @@ android {
     buildTypes {
         release {
```
Comment (TalhaAli00) [inline androidCore/sample/build.gradle.kts:20]: Maybe we don't need this sample. I think we can get rid of it

File: sign/sdk/build.gradle.kts [hunk 3]
```diff
@@ -65,7 +65,6 @@ dependencies {
     debugImplementation(project(":androidCore:sdk"))
     releaseImplementation("com.walletconnect:android-core:$CORE_VERSION")
 
-    moshiKsp()
```
Comment (TalhaAli00) [inline sign/sdk/build.gradle.kts:68]: Why was this removed?

**Review by TalhaAli00 — APPROVED** [review]

File: androidCore/sample/build.gradle.kts (hunk 2 above)
Comment (jakobuid) [reply androidCore/sample/build.gradle.kts:20]: Yea that's true we could

File: sign/sdk/build.gradle.kts (hunk 3 above)
Comment (jakobuid) [reply sign/sdk/build.gradle.kts:68]: Because while testing sdks with updated dependencies it seemed to be unnecessary here
